lib/diagnostic.rb

```diff
@@ -45,12 +55,12 @@
 # What is the type of object in Ruby for decimal numbers? Write your answer in
 # a Ruby string.
 
-Diagnostic.decimal = 'your response here'
+Diagnostic.decimal = 'float'
 
 # What is the type of object in Ruby for integer numbers? Write your answer in
 # a Ruby string.
 
-Diagnostic.integer = 'your response here'
+Diagnostic.integer = 'interger'
 
 # Write an example of a decimal and an integer in Ruby. Replace the Ruby strings
 # in the following array with your examples.
```

Comment:
Thank you I looked back over these and was able to see my error for the
predicate problem and complete most of these without issue, except for the
hash questions. Those are still not quite clicking

On Wednesday, August 3, 2016, Lauren Fazah <notifications@github.com> wrote:

> In lib/diagnostic.rb
> <https://github.com/ga-wdi-boston/ruby-diagnostic/pull/97#discussion_r73349529>
> :
>
> >
> >  # What is the type of object in Ruby for integer numbers? Write your answer in
> >  # a Ruby string.
> >
> > -Diagnostic.integer = 'your response here'
> > +Diagnostic.integer = 'interger'
> >
> >  # Write an example of a decimal and an integer in Ruby. Replace the Ruby strings
> >  # in the following array with your examples.
>
> numbers - [1.0, 1]
> falsy - [nil, false]
> flow_control - "The Dark Night"
> else_if - 'elsif'
> returns - false
> remove_from_array - arr.pop 2
> array_add_to - [12, 34, 56, 67, nil, nil, 9]
> person_hash - person = {first_name: "Natalia", age: 88}
> hash_default - person.default = \"\#{person[:first_name]}
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ga-wdi-boston/ruby-diagnostic/pull/97/files/15baba5ce2ffaa00ff3ef8f2745d1f3f11aa0d3f#r73349529>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARzm6Gjh6fCrrMTHyY1jW9ylzZiGDHbPks5qcLKYgaJpZM4JbqSF>
> .
>


-- 
Sent from Gmail Mobile